package/src/main.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022 Oxide Computer Company
+
+use anyhow::Result;
+use std::fs::create_dir_all;
+use std::path::Path;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    let cfg = omicron_zone_package::config::parse("package-manifest.toml")?;
+
+    let output_dir = Path::new("out");
+    create_dir_all(&output_dir)?;
+
+    for package in cfg.packages.values() {
+        package.create(output_dir).await?;
+    }
```

Comment:
Do we expect to include multiple packages in propolis' manifest?

Reply:
I think at the moment we're just expecting the single package out of propolis - iterating over all packages in the manifest is a bit future-proofy.